stylesheet.css

```diff
@@ -0,0 +1,51 @@
+@font-face {
+    font-family: "PokemonHollow";
+    src: url(PokemonHollow.ttf);
+}
+
+@font-face {
+    font-family: "PokemonSolid";
+    src: url(PokemonSolid.ttf);
+}
+
+h1 {
+    font-family: "PokemonSolid";
+    color: black;
```

Comment:
its better practice to specify the colour without using its name, eg hex colours